### DISCUSSION TITLE
Support `enum` in Entity fields.

### DISCUSSION THREAD

**Discussion by dkaera** [vitusortner/floor#672] [Ideas]
Hey guys, how about adding `enum` support to Entity fields without additional type converters?

**Comment by ThomasMiddel** [comment]
Sounds like a very nice addition!

**Reply by dkaera** [reply]
alright
since it should take less time than supporting raw queries, I want to do it first.

**Reply by ThomasMiddel** [reply]
Awesome @dkaera 💪

**Reply by dkaera** [reply]
Hey @ThomasMiddel 
Just pushed the enum support integration [here](https://github.com/pinchbv/floor/pull/681)

**Reply by ThomasMiddel** [reply]
Niiiice, I'll have a look ASAP!

**Reply by dkaera** [reply]
Hey @ThomasMiddel how're things?
Do you have any comments or concerns about the integration of enumeration processing?

**Comment by dkaera** [comment]
Merged to develop #681
